data/sql/updates/pending_db_world/rev_1487514602617381300.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1487514602617381300');
+-- Table `item_template`
+UPDATE `item_template` SET `spellcooldown_1` = 30000 WHERE `entry` IN (44010);
```

Comment:
just a not for future: please add an empty line at the end of file, it will help to avoid problems when more SQL files get combined